tests/TestDataService.spec.ts

```diff
@@ -23,13 +23,55 @@ test('Data Service', async ({
     expect(customer.firstName).toEqual('Luke');
     expect(customer.lastName).toEqual('Skywalker');
 
+    const customerAddress = await TestDataService.getCustomerAddress(customer.defaultBillingAddressId);
+    const deliveryStateMachine = await TestDataService.getDeliveryStateMachine();
+    const deliveryState = await TestDataService.getStateMachineState(deliveryStateMachine.id);
+    const shippingMethod = await TestDataService.getShippingMethod();
+    const deliveryStruct = TestDataService.getBasicOrderDeliveryStruct(deliveryState, shippingMethod, customerAddress);
+
+    const shippingCosts = 5;
+    const totalPrice = 50+ shippingCosts;
+
+    if (deliveryStruct.shippingCosts != null) {
+        deliveryStruct.shippingCosts.unitPrice = shippingCosts;
+        deliveryStruct.shippingCosts.totalPrice = shippingCosts;
+    }
+
+    const customShippingCosts = {
+        price: {
+            totalPrice: totalPrice,
+            positionPrice: totalPrice,
+            rawTotal: totalPrice,
+            netPrice: totalPrice,
+            taxStatus: 'gross',
+            calculatedTaxes: [{
+                tax: 0,
+                taxRate: 0,
+                price: totalPrice,
+            }],
+            taxRules: [{
+                taxRate: 0,
+                percentage: 100,
+            }],
+        },
+        deliveries: [deliveryStruct],
+    };
+
     const order = await TestDataService.createOrder(
         [{ product, quantity: 5 }],
         customer,
-        { orderNumber: '123456789' });
-    expect(order.orderNumber).toEqual('123456789');
-    expect(order.orderCustomer.firstName).toEqual('Luke');
-    expect(order.price.totalPrice).toEqual(58.99);
+        customShippingCosts,
+        );
+    expect(order.price.totalPrice).toEqual(55);
+
+    const order2 = await TestDataService.createOrder(
```

Comment:
Magic numbers are always a bit fuzzy. A defined naming would be better like `orderWithCustomShipping` or something.